app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,80 @@
+<nav id="navcontainer" class="navbar navbar-default navbar-fixed-top">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <div id="date" class="navbar-brand">
+        <% if @count -%>
+        <span id="badge_count" class="number_badge"><%= @count %></span>
+        <% end -%>
+        <%= l(Time.zone.today, :format => current_user.prefs.title_date_format) %>
+      </div>
+    </div>
+    <div id="navbar" class="navbar-collapse collapse" aria-expanded="false">
```

Comment:
`aria-expanded` is not, however, needed on the target element. In the Bootstrap docs it's only placed on the controlling element.